TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/PageSwipeTest.kt

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.twopage
+
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.test.performGesture
+import androidx.compose.ui.test.swipeLeft
+import androidx.compose.ui.test.swipeRight
+import androidx.window.layout.WindowMetricsCalculator
```

Comment:
Do you think that we should include `implementation("androidx.window:window-testing:1.0.0-rc01")` into our `testUtil` or `WindowState` library? Since we don't talk to JWM in the sample, I wonder whether we should avoid talking to JWM testing in our sample test as well.